notebooks/test_output/MRSA/2017-08-04_MRSA_1-4_sample_sheet.csv

```diff
@@ -8,7 +8,7 @@ Experiment Name,RKL_experiment
 Date,2017-08-13
 Workflow,GenerateFASTQ
 Application,FASTQ Only
-Assay,Metagenomic
+Assay,Metagenomics
```

Comment:
weirdly, this sample_sheet has Metagenomics when it should be Metagenomic

Reply:
I believe this file was removed.